name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/dataseries/DataSeriesSerializer.java

```diff
@@ -49,12 +49,16 @@ private void load(DataSeriesSet set, String config, List<DataSeries> series)
             {
                 series.add(s);
 
-                if (data.length == 4)
+                if (data.length >=4 && data.length <=5)
```

Comment:
Shouldn't the condition ``data.length >= 4`` be sufficiant?
Then it's quite easier to add more data items in the future without checking the old conditions. The condition can remain as long as data fields are added.

Reply:
Thanks for the hint. I agree.